pkg/config/config.go

```diff
@@ -76,30 +76,47 @@ func LoadIPAMConfig(bytes []byte, envArgs string, extraConfigPaths ...string) (*
 		logging.Debugf("Used defaults from parsed flat file config @ %s", foundflatfile)
 	}
 
-	if r := strings.SplitN(n.IPAM.Range, "-", 2); len(r) == 2 {
-		firstip := netutils.ParseIPSloppy(r[0])
-		if firstip == nil {
-			return nil, "", fmt.Errorf("invalid range start IP: %s", r[0])
-		}
-		lastip, ipNet, err := netutils.ParseCIDRSloppy(r[1])
-		if err != nil {
-			return nil, "", fmt.Errorf("invalid CIDR (do you have the 'range' parameter set for Whereabouts?) '%s': %s", r[1], err)
-		}
-		if !ipNet.Contains(firstip) {
-			return nil, "", fmt.Errorf("invalid range start for CIDR %s: %s", ipNet.String(), firstip)
-		}
-		n.IPAM.Range = ipNet.String()
-		n.IPAM.RangeStart = firstip
-		n.IPAM.RangeEnd = lastip
-	} else {
-		firstip, ipNet, err := netutils.ParseCIDRSloppy(n.IPAM.Range)
-		if err != nil {
-			return nil, "", fmt.Errorf("invalid CIDR %s: %s", n.IPAM.Range, err)
-		}
-		n.IPAM.Range = ipNet.String()
-		if n.IPAM.RangeStart == nil {
-			firstip = netutils.ParseIPSloppy(firstip.Mask(ipNet.Mask).String()) // if range_start is not net then pick the first network address
-			n.IPAM.RangeStart = firstip
+  if n.IPAM.Range != "" {
+
+    logging.Debugf("Deprecated Range present in IPAM config. It will soon be removed in later releases. Please use IPRanges instead")
```

Comment:
I'm not sure about actually deprecating this. I think I like having a single range still be an option, and it's simpler to type in. However, at debugf logging level, I think I'm OK with this comment. (no change necessary, thinking out loud)

Reply:
> I'm not sure about actually deprecating this. I think I like having a single range still be an option, and it's simpler to type in. However, at debugf logging level, I think I'm OK with this comment. (no change necessary, thinking out loud)

@dougbtv Thank you so much for reviewing :smiley: